package.json

```diff
@@ -1,39 +1,16 @@
 {
   "name": "loopback-next",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/strongloop/loopback-next.git"
-  },
+  "description": "LoopBack makes it easy to build modern API applications that require complex integrations",
```

Comment:
Maybe a better description:

`Monorepo for LoopBack 4 - A highly extensible Node.js and TypeScript framework for building APIs and microservices.`

Reply:
Done